takwimu/takwimu_ui/src/components/FeaturedData.js

```diff
@@ -18,40 +18,61 @@ const styles = () => ({
   }
 });
 
-function FeaturedData({ classes, takwimu: { featured_data: featuredData } }) {
+function FeaturedData({
+  classes,
+  takwimu: {
+    page: {
+      featured_data: { value: featuredData }
+    }
+  }
+}) {
+  if (!featuredData) {
+    return null;
+  }
+
+  const { title, featured_data: featuredDataWidgets } = featuredData;
   return (
-    <Section title="Featured Data" variant="h2">
-      {featuredData && featuredData.length ? (
+    <Section title={title} variant="h2">
+      {featuredDataWidgets && featuredDataWidgets.length > 0 && (
         <Grid
           container
           direction="row"
           justify="space-between"
           alignItems="flex-start"
           className={classes.root}
         >
-          <DataContainer color="secondary" data={featuredData[0]} />
-          <DataContainer color="primary" data={featuredData[1]} />
+          <DataContainer color="secondary" data={featuredDataWidgets[0]} />
+          {featuredDataWidgets.length > 1 && (
```

Comment:
`{featuredDataWidgets.length > 1 && (` use `[].slice(0,2).map(...)`

then remove `{featuredDataWidgets && featuredDataWidgets.length > 0 && (`

Reply:
Different cases: if you have one `featuredDataWidgets`, show it. If you have more than one, just show the second one, ignore the rest.

Reply:
`.slice(0,2)` handles that.

Reply:
Yeah and that's where I need to make the change then... No need to `slice` just use `featuredDataWidgets[1]`